modules/calib3d/src/five-point.cpp

```diff
@@ -401,6 +401,29 @@ class EMEstimatorCallback CV_FINAL : public PointSetRegistrator::Callback
     }
 };
 
+// Find essential matrix given undistorted points and two cameras.
+static Mat findEssentialMat( InputArray _points1, InputArray _points2,
```

Comment:
Consider adding underscore `_` suffix for internal names to avoid confusion with public API: `findEssentialMat_`